medusa/failed_processor.py

```diff
@@ -52,30 +44,30 @@ def process(self):
             parsed = NameParser().parse(releaseName)
         except (InvalidNameException, InvalidShowException):
             self._log(u'Not enough information to parse release name into a valid show. '
-                      u'Consider adding scene exceptions or improve naming for: {release}'.format
-                      (release=releaseName), logger.WARNING)
+                      u'Consider adding scene exceptions or improve naming for: {release}',
+                      {'release': releaseName}, logger.WARNING)
             raise FailedPostProcessingFailedException()
 
-        self._log(u'Parsed info: {result}'.format(result=parsed), logger.DEBUG)
+        self._log(u'Parsed info: {result}', {'result': parsed}, logger.DEBUG)
```

Comment:
I don't need to use brace here as I only need to pass the `message` to self._log